site/assets/css/fish_style.css

```diff
@@ -263,9 +263,7 @@ h3 {
     image-rendering: -o-crisp-edges;
     image-rendering: -webkit-optimize-contrast;
     image-rendering: -webkit-crisp-edges;
-    image-rendering: pixelated;
-    image-rendering: optimize-contrast; 
-    image-rendering: crisp-edges;
+    image-rendering: optimize-contrast;
```

Comment:
(`crisp-edges` is the ideal filtering (for scaling pixel art, like a SNES emulator might. It's also the newest value to be standardized so this change now effectively is also overwriting `-webkit-crisp-edges`and `-o-crisp-edges` anyhow.